vnext/Desktop/WebSocket.h

```diff
@@ -109,4 +120,99 @@ class SecureWebSocket : public BaseWebSocket<Protocol, Socket, Resolver>
   SecureWebSocket(Url&& url);
 };
 
-} } // namespace facebook::react
+namespace Test {
```

Comment:
This should be in a sperate file. also we should only includes them in test project

Reply:
@licanhua I agree in principle, but this would yield linking errors, due to the templated nature of `BaseWebSocket` when linking `WebSocket.cpp`. In order to avoid circular dependencies on headers, I decided to not write these declarations in the unit test project.

That said, we could define a macro that builds the test variants only when needed.

Reply:
You may add some comments on it, and explain why you need test in this file.
Also It's better the end the Microsoft::react namespace first, then start your new namespace. eg:

} } // namespace Microsoft::react
namespace Microsoft::react::Test {

---
In reply to: [281372647](https://github.com/microsoft/react-native-windows/pull/2415#discussion_r281372647) [](ancestors = 281372647)